cmd/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/mainflux/callhome/callhome"
+	"github.com/mainflux/callhome/callhome/api"
+	"github.com/mainflux/callhome/callhome/repository/sheets"
+	"github.com/mainflux/callhome/callhome/repository/timescale"
+	"github.com/mainflux/callhome/internal"
+	jaegerClient "github.com/mainflux/callhome/internal/clients/jaeger"
+	"github.com/mainflux/callhome/internal/clients/postgres"
+	"github.com/mainflux/callhome/internal/env"
+	"github.com/mainflux/callhome/internal/server"
+	httpserver "github.com/mainflux/callhome/internal/server/http"
+	mflog "github.com/mainflux/mainflux/logger"
+	"golang.org/x/sync/errgroup"
+)
+
+const (
+	svcName        = "callhome"
+	envPrefix      = "MF_CALLHOME_"
+	envPrefixHttp  = "MF_CALLHOME_"
+	defSvcHttpPort = "8855"
+)
+
+type config struct {
+	LogLevel       string `env:"MF_CALLHOME_LOG_LEVEL"       envDefault:"info"`
+	JaegerURL      string `env:"MF_JAEGER_URL"               envDefault:"localhost:6831"`
+	GCPCredFile    string `env:"MF_CALLHOME_GCP_CRED"`
+	SpreadsheetId  string `env:"MF_CALLHOME_SPREADSHEET_ID"`
+	SheetId        int    `env:"MF_CALLHOME_SHEET_ID"        envDefault:"0"`
+	IPDatabaseFile string `env:"MF_CALLHOME_IP_DB"           envDefault:"IP2LOCATION-LITE-DB5.BIN"`
+}
```

Comment:
I agree with you idea of keeping `IP2LOCATION-LITE-DB5.BIN` in root of repo, So please move back the file to repo root directory 

Instead of having abosulte path , I suggest you to have relative path  `./IP2LOCATION-LITE-DB5.BIN`  in environmental variable default value
```suggestion
	IPDatabaseFile string `env:"MF_CALLHOME_IP_DB"           envDefault:"./IP2LOCATION-LITE-DB5.BIN"`
```